SEED-platform/seed
# Allow analysis property view related properties to be null

#### Any background context you want to provide?
Analyses have a many-to-many relationship with Properties and PropertyStates via a join table AnalysisPropertyView. If a property is deleted, its Analyses and AnalysisPropertyViews are cascade deleted. 

#### What's this PR do?
This PR allows AnalysisPropertyView to have null values for both Property and PropertyState. If an analysis is related to a deleted property, the property appears as a warning icon + 'Unknown'. There is an existing tooltip that will show 'Property no longer exists' when hovering over the unknown property. This PR also removes the checkboxes 'Also delete x analyses' and 'Also delete x batch analyses' from the property delete modal.

#### How should this be manually tested?
Upload properties, create analyses, delete some of those properties, and check the analyses.

#### What are the relevant tickets?
#4218

#### Screenshots (if appropriate)
![Screenshot 2023-09-19 at 9 56 28 AM](https://github.com/SEED-platform/seed/assets/58446472/d440f1ed-9790-4cad-a403-2e6d8dc62818)

![Screenshot 2023-09-19 at 9 55 12 AM](https://github.com/SEED-platform/seed/assets/58446472/350e4c89-be5d-4624-8274-471d9676d1c3)

![Screenshot 2023-09-19 at 9 55 44 AM](https://github.com/SEED-platform/seed/assets/58446472/cf3037e6-6e3c-41a9-b4ad-06c510574c41)

![Screenshot 2023-09-19 at 9 55 55 AM](https://github.com/SEED-platform/seed/assets/58446472/7fc8788b-cfb3-4766-b5a5-c0ed38ac68d7)
